graphix/sim/statevec.py

```diff
@@ -114,7 +114,7 @@ def __init__(
 
         else:
             if isinstance(input_list[0], graphix.states.State):
-                graphix.types.check_list_elements(input_list, graphix.states.State)
+                type_utils.check_list_elements(input_list, graphix.states.State)
```

Comment:
I am not sure we need to do that anymore (it is somehow redundant with static type checking).

Reply:
Sadly we need it until we annotate `statevec.py`.